var/spack/repos/builtin/packages/biobb-common/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class BiobbCommon(PythonPackage):
+    """Biobb_common is the base package required to use the biobb packages"""
+
+    homepage = "https://github.com/bioexcel/biobb_common"
+    git = "https://github.com/bioexcel/biobb_common.git"
```

Comment:
It's best to download stable releases from PyPI: https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#downloading

Reply:
done